app/views/products/index.html.erb

```diff
@@ -0,0 +1,80 @@
+<%= form_tag products_category_path, method: "get" do %>
+  <%= select_tag "category", options_from_collection_for_select(Category.all, "id", "name") %>
+  <%= submit_tag "Sort by Category" %>
+<% end %>
+
+<%= form_tag products_merchant_path, method: "get" do %>
+  <%= select_tag "user", options_from_collection_for_select(User.all, "id", "username") %>
+  <%= submit_tag "Sort by Merchant" %>
+<% end %>
+
+<% if request.path.include?("filter") %>
+  <%= link_to "See all Products", products_path, method: :get, class: "btn" %>
+<% end %>
+
+
+
+<h1>All Products</h1>
+<div class = "container-fluid">
+  <div class ="row">
+    <div class = "col-sm-4">
+      <h1>Name</h1>
+    </div>
+    <div class = "col-sm-4">
+      <h1>Photo</h1>
+    </div>
+    <div class = "col-sm-4">
+      <h1>Price</h1>
+    </div>
+  </div>
+  <% if @products_array.empty? && @merchant_products.nil? %>
+    <% @products.each do |product| %>
```

Comment:
Since the logic which displays these section always checks that the products are active, I think it would make more sense to only populate these two lists: `@products_array` and `@merchant_products` with active products 

(note this would negate an earlier comment i suggested which was to remove the `@merchant_products` instance variable because it wasn't adding any specific value)